prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -109,10 +109,29 @@ periodics:
 # kyma-integration-cleaner removes all sshPublic keys stored for service account "sa-vm-kyma-integration". Those keys refers to machines that in most cases were already removed.
 # Purpose of this job is to avoid error: "Login profile size exceeds 32 KiB" when running kyma-integration tests
 - name: utilities-kyma-integration-cleaner
-  cron: "0 7 * * 1-5" # “At 07:00 on every day-of-week from Monday through Friday.”
+  cron: "0 7 * * 1-5" # "At 07:00 on every day-of-week from Monday through Friday."
   labels:
     preset-sa-vm-kyma-integration: "true"
     preset-gc-project-env: "true"
   spec:
     containers:
     - image: eu.gcr.io/kyma-project/prow/cleaner:0.0.1 # see test-infra/prow/images/cleaner
+- name: orphanded-disks-cleaner
+  cron: "0 * * * *" # "Every hour"
+  labels:
+    preset-sa-gke-kyma-integration: "true"
+    preset-gc-project-env: "true"
+  decorate: true
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+    - image: eu.gcr.io/kyma-project/prow/buildpack-golang:0.0.1
+      command:
+        - "bash"
+      args:
+        - "-c"
+        - "development/disks-cleanup.sh ${CLOUDSDK_CORE_PROJECT} --dryRun=false"
```

Comment:
disks-cleanup.sh script seems to accept but one argument, so the "--dryRun=true" flag is not going to affect the way it works, I guess.

Reply:
You're right. It's redundant.